### PR TITLE
Olly + Carly: Making Question

### PR DESCRIPTION
**Pull request by olly-larkin**

### CODE REVIEW

File: src/component/AnswerComp.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Answer } from "model/Answer";
+import styled from "styled-components";
+import "css/fonts.css";
+
+interface AnswerContentProps {
+  answer: Answer;
+}
+
+const AnswerContentContainer = styled.div`
```
Comment (JoshAdamPowell) [inline src/component/AnswerComp.tsx]: I think we should have the styled components in their own file. In the climate-change-app we each component has an associated `styled.ts` file where we export them

File: src/component/AnswerComp.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  );
+};
+
+interface AnswerCompProps {
+  answer: Answer;
+}
+
+const AnswerButton = styled.button`
+  background: #41bcec;
+  border: 2px solid #259bbf;
```
Comment (JoshAdamPowell) [inline src/component/AnswerComp.tsx]: we should probably have all our colours in one place

File: src/component/AnswersComp.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  answers: Answer[];
+}
+
+const AnswersContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 10px 10px;
+`;
+
+export const AnswersComp = ({ answers }: AnswersCompProps) => (
```
Comment (JoshAdamPowell) [inline src/component/AnswersComp.tsx]: I think I'm just being dumb but what is a Comp?

File: src/component/AnswersComp.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import { Answer } from "model/Answer";
+import { AnswerComp } from "component/AnswerComp";
+import styled from "styled-components";
+
+interface AnswersCompProps {
+  answers: Answer[];
+}
+
+const AnswersContainer = styled.div`
+  display: grid;
```
Comment (JoshAdamPowell) [inline src/component/AnswersComp.tsx]: this is setting off my 🚨wont work in IE alarm🚨. I haven't tested it though

Comment (JoshAdamPowell) [reply src/component/AnswersComp.tsx]: oh thats fine we explicitly dont support IE, nice.

File: src/component/MultiChoiceQComp.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import { Question } from "model/Question";
+import { AnswersComp } from "component/AnswersComp";
+import styled from "styled-components";
+import "css/fonts.css";
+
+interface MultiChoiceQProps {
+  question: Question;
+}
+
+const MultiChoiceQContainer = styled.div`
```
Comment (JoshAdamPowell) [inline src/component/MultiChoiceQComp.tsx]: ditto into pulling these into a `styled.ts`

File: src/component/AnswerComp.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+      )}
+    </AnswerContentContainer>
+  );
+};
+
+interface AnswerCompProps {
+  answer: Answer;
+}
+
+const AnswerButton = styled.button`
```
Comment (JoshAdamPowell) [inline src/component/AnswerComp.tsx]: can we have `cursor: pointer` here please

**Review by JoshAdamPowell — CHANGES_REQUESTED** [review]

File: src/component/AnswersComp.tsx (hunk 3 above)
Comment (olly-larkin) [reply src/component/AnswersComp.tsx]: This is just short for component

Comment (olly-larkin) [reply src/component/AnswersComp.tsx]: If you have a better name I'm definitely up for changing it though

Comment (JoshAdamPowell) [reply src/component/AnswersComp.tsx]: hm maybe something like `AnswerSquare`? maybe even just `Answer`

File: src/component/AnswersComp.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  grid-template-columns: 1fr 1fr;
+  gap: 10px 10px;
+`;
+
+export const AnswersComp = ({ answers }: AnswersCompProps) => (
+  <AnswersContainer>
+    {answers.map((answer) => (
+      <AnswerComp answer={answer} />
```
Comment (JoshAdamPowell) [inline src/component/AnswersComp.tsx]: Sorry I only just noticed, I this this needs a `key` prop

**Review by JoshAdamPowell — APPROVED** [review]
nice!

**Pull request merged**